README.md

```diff
@@ -1,18 +1,45 @@
-# About the column pinning effort
+# SlickGrid (Coatue)
```

Comment:
Better to read this file rather than look at the diff.

All I did here was move the original introduction back to the top, add a notice about this fork vs. the original, and broke lines so each line fits in 120 characters.